components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__gender.xml

```diff
@@ -111,6 +111,9 @@
       <content>{{include reference="PhenoTips.PatientSheetMacros" /}}
 
 {{velocity}}
+#if ($xcontext.action == 'edit' &amp;&amp; $obj.getProperty('gender').value == '')
```

Comment:
Right, I see. Now it worked just because we first save an empty patient when clicking on "Create New", but since we are going to fix it soon (assigned to me :) ) this fix needs to be more generic.